src/modules/Builder/features/ActivitySettings/TitleComponent/TitleComponent.tsx

```diff
@@ -24,6 +27,10 @@ export const TitleComponent = ({ title, name, open }: TitleComponentProps) => {
     : [];
   const isShowErrors = !open && errorMessages.length > 0;
 
+  useEffect(() => {
+    trigger(name);
+  }, [entities.length]);
```

Comment:
Could you please modify the useEffect to include dependencies, or if necessary, disable the dependency check for useEffect?